src/eko/thresholds.py

```diff
@@ -223,3 +253,48 @@ def nf(self, q2):
         """
         ref_idx = np.digitize(q2, self.area_walls)
         return 2 + ref_idx
+
+    def compute_msbar_mass(
+        self,
+    ):
+        """
+        Compute the MSBar masses solving the equation :math:`m_{\bar{MS}}(m) = m`
+        """  # pylint:disable=import-outside-toplevel
+        from .msbar_masses import evolve_msbar_mass
```

Comment:
Why do you need to import outside toplevel?

Reply:
Otherwise there is a cyclic import with `StrongCoupling`, `ThresholdAtlas`, `msbar_masses` ... I need to find a better solution if possible

Reply:
Maybe simply don't put `compute_msbar_masses` in here: make a stand-alone function inside `msbar_masses` module, call it inside `Runner` and pass the reference values to threshold